interface/src/Application.cpp

```diff
@@ -7069,19 +7081,23 @@ void Application::updateWindowTitle() const {
 
     auto nodeList = DependencyManager::get<NodeList>();
     auto accountManager = DependencyManager::get<AccountManager>();
+    auto domainAccountManager = DependencyManager::get<DomainAccountManager>();
     auto isInErrorState = nodeList->getDomainHandler().isInErrorState();
+    bool isMetaverseLoggedIn = accountManager->isLoggedIn();
+    bool isDomainLoggedIn = domainAccountManager->isLoggedIn();
+    QString authedDomain = domainAccountManager->getAuthedDomain();
 
     QString buildVersion = " - Vircadia - "
         + (BuildInfo::BUILD_TYPE == BuildInfo::BuildType::Stable ? QString("Version") : QString("Build"))
         + " " + applicationVersion();
 
-    QString loginStatus = accountManager->isLoggedIn() ? "" : " (NOT LOGGED IN)";
-
     QString connectionStatus = isInErrorState ? " (ERROR CONNECTING)" :
         nodeList->getDomainHandler().isConnected() ? "" : " (NOT CONNECTED)";
-    QString username = accountManager->getAccountInfo().getUsername();
 
-    setCrashAnnotation("sentry[user][username]", username.toStdString());
+    QString metaverseUsername = accountManager->getAccountInfo().getUsername();
+    QString domainUsername = domainAccountManager->getUsername();
+
+    setCrashAnnotation("sentry[user][metaverseUsername]", metaverseUsername.toStdString());
```

Comment:
changing the attribute from [username] to [metaverseUsername] may be an issue here, [username] was chosen here as it is a recognized keyword that is interpreted in Sentry as per https://develop.sentry.dev/sdk/event-payloads/user/

Reply:
Good point, we can just use secondary metadata later to explain a domain user and assume the master "user" is a metaverse one. :)

Reply:
If you do make up your own tags then replace the entire field and don't start it with "sentry[", that way they'll end up at the bottom of any error reports.  Unrecognized keywords in the "sentry" namespace appear to be eaten.

Reply:
Good catch! Will revert to using "[username]".